src/fides/api/util/consent_util.py

```diff
@@ -455,16 +466,14 @@ def prepare_privacy_notice_patches(
 
     # run the validation here on our proposed "dry-run" updates
     try:
-        check_conflicting_data_uses(
-            validation_updates,
-            existing_notices.values(),
-            ignore_disabled=ignore_disabled,
-        )
```

Comment:
FYI This same file has the code that prefers opt out preferences in case of conflict for consent propagation https://github.com/ethyca/fides/blob/main/src/fides/api/util/consent_util.py#L97!